### PR TITLE
Poll withdraw/deposit flows and update UI on completion

### PR DESCRIPTION
**Pull request by piyalbasu**
Much of the work in this PR is bringing the steps of the SEP-6 flow closer to the SEP-24 flow

### CODE REVIEW

File: src/components/Sep6/Sep6Withdraw.tsx [hunk 1]
```diff
@@ -225,21 +236,44 @@ export const Sep6Withdraw = () => {
...
         <div className="ModalBody">
           <InfoBlock>
-            <strong>Account ID: </strong>
+            <strong>Sending Payment To: </strong>
 
-            {withdrawResponse.account_id}
+            {shortenStellarKey(withdrawResponse.account_id)}
           </InfoBlock>
 
+          <Input
```
Comment (piyalbasu) [inline src/components/Sep6/Sep6Withdraw.tsx:251]: Making a slight adjustment to the Withdraw flow to allow user to submit payment to anchor from UI

File: src/components/Sep6/Sep6Withdraw.tsx [hunk 2]
```diff
@@ -262,20 +296,50 @@ export const Sep6Withdraw = () => {
...
-            <InfoBlock>
-              <strong>Max Amount: </strong>
+  if (sep6WithdrawAsset.status === ActionStatus.SUCCESS) {
+    return (
+      <Modal visible={true} onClose={handleClose}>
+        <Heading2 className="ModalHeading">SEP-6 Withdrawal Completed</Heading2>
 
-              {withdrawResponse.max_amount}
+        <div className="ModalBody">
+          {transactionResponse.to && (
```
Comment (piyalbasu) [inline src/components/Sep6/Sep6Withdraw.tsx:313]: And show the user transaction results, as recommended by SEP-6

File: src/ducks/sep6DepositAsset.ts [hunk 3]
```diff
@@ -209,37 +232,63 @@ export const submitSep6DepositFields = createAsyncThunk<
...
-        type,
-      } = sep6data;
+      } = sep6Data;
 
-      const depositResponse = (await programmaticDepositFlow({
-        assetCode,
-        publicKey,
-        transferServerUrl: transferServer,
+      const trustAssetCallback = async () => {
+        const assetString = `${assetCode}:${assetIssuer}`;
```
Comment (piyalbasu) [inline src/ducks/sep6DepositAsset.ts:260]: follow the `trustAsset` and then `pollUntilComplete` flow that SEP-24 uses

File: src/components/Assets.tsx [hunk 4]
```diff
@@ -228,7 +232,55 @@ export const Assets = ({
     history,
   ]);
 
-  // Deposit asset
+  // SEP-6 Deposit asset
+  useEffect(() => {
```
Comment (piyalbasu) [inline src/components/Assets.tsx:236]: reload UI upon deposit/withdrawal completion

File: src/components/Sep6/Sep6Withdraw.tsx [hunk 5]
```diff
@@ -219,27 +230,50 @@ export const Sep6Withdraw = () => {
...
 
-  if (sep6WithdrawAsset.status === ActionStatus.SUCCESS) {
+  if (sep6WithdrawAsset.status === ActionStatus.CAN_PROCEED) {
     return (
       <Modal visible={true} onClose={handleClose}>
-        <Heading2 className="ModalHeading">SEP-6 Withdraw Info</Heading2>
+        <Heading2 className="ModalHeading">Payment Sending</Heading2>
 
         <div className="ModalBody">
           <InfoBlock>
```
Comment (quietbits) [inline src/components/Sep6/Sep6Withdraw.tsx:245]: Just a heads up that `InfoBlock` UI will change. Maybe we could use a "simple" confirmation layout here?

image

File: src/components/UntrustedBalance.tsx [hunk 6]
```diff
@@ -97,12 +102,23 @@ export const UntrustedBalance = ({
       balance: asset,
     };
 
+    const TRUSTED_ASSET_EXPLANATION =
+      "A lumen is the only asset type that can be used on the Stellar network that doesn’t require an issuer or a trustline.";
```
Comment (quietbits) [inline src/components/UntrustedBalance.tsx:106]: 👏

**Review by quietbits — APPROVED** [review]

File: src/components/Sep6/Sep6Withdraw.tsx (hunk 5 above)
Comment (piyalbasu) [reply src/components/Sep6/Sep6Withdraw.tsx:245]: Oh, thanks for the heads up! I'll use a different element here

**Pull request merged**